### DISCUSSION TITLE
Create App For UmbrelOS

### DISCUSSION THREAD

**Discussion by anieto** [AdguardTeam/AdGuardHome#3763] [General]
Recently setup my own Bitcoin Lightning Node using [UmbrelOS](https://getumbrel.com). It allows installing apps like PiHole but I'd prefer to use AdGuard Home. Any chance the team could create and submit a build for UmbrelOS? They utilize docker which I know AdGuard already provides an image for. It's a little too far out of my wheelhouse to get up and running.

https://github.com/getumbrel/umbrel/blob/master/apps/README.md

**Comment by buttonbeaver** [comment]
Looks like someone is working on it here: https://github.com/getumbrel/umbrel-apps/pull/603
